packages/gestalt/src/FormElement.css

```diff
@@ -0,0 +1,33 @@
+.base {
+  appearance: none;
+  border-radius: 16px;
+  border-style: solid;
+  border-width: 2px;
```

Comment:
These are all using 2px borders so seems like another argument for adding support for 2px border width when I implement borders for Box